host-spec/c06-consensus.tm

```diff
@@ -1052,27 +1052,51 @@
+    refers to the signature of a voter for a specific message in a round and
+    is formally defined as:
+
+    <\equation*>
+      Sign<rsup|r,stage><rsub|v<rsub|i>>=Sig<rsub|ED25519><around*|(|Msg,r,id<rsub|\<bbb-V\>>|)>
+    </equation*>
+
+    Where:
+
+    <\big-table|<tabular|<tformat|<cwith|2|3|1|1|cell-halign|r>|<cwith|2|3|1|1|cell-lborder|0ln>|<cwith|2|3|2|2|cell-halign|l>|<cwith|2|3|3|3|cell-halign|l>|<cwith|2|3|3|3|cell-rborder|0ln>|<cwith|2|3|1|3|cell-valign|c>|<table|<row|<cell|Msg>|<cell|the
+    message to be signed>|<cell|arbitrary>>|<row|<cell|r:>|<cell|round
+    number>|<cell|unsigned 64-bit integer>>|<row|<cell|<math|id<rsub|\<bbb-V\>>>>|<cell|authority
+    set Id (Def. <reference|defn-authority-set-id>) of v>|<cell|unsigned
+    64-bit integer>>>>>>
+      Signature for a message in a round.
+    </big-table>
+
+    \;
+  </definition>
+
   <\definition>
     A vote casted by voter <math|v> should be broadcasted as a
     <strong|message <math|M<rsup|r,stage><rsub|v>>> to the network by voter
     <math|v> with the following structure:
 
     <\eqnarray*>
-      <tformat|<table|<row|<cell|M<rsup|r,stage><rsub|v>>|<cell|\<assign\>>|<cell|Enc<rsub|SC><around*|(|r,id<rsub|\<bbb-V\>>,<math-it|SigMsg>|)>>>|<row|<cell|<math-it|SigMsg>>|<cell|\<assign\>>|<cell|<math-it|Msg>,Sig<rsub|ED25519><around*|(|<math-it|Msg>,r,id<rsub|\<bbb-V\>>|)>,v<rsub|id>>>|<row|<cell|<math-it|Msg>>|<cell|\<assign\>>|<cell|Enc<rsub|SC><around*|(|stage,V<rsup|r,stage><rsub|v>|)>>>>>
+      <tformat|<table|<row|<cell|M<rsup|r,stage><rsub|v>>|<cell|\<assign\>>|<cell|Enc<rsub|SC><around*|(|r,id<rsub|\<bbb-V\>>,<math-it|SigMsg>|)>>>|<row|<cell|<math-it|SigMsg>>|<cell|\<assign\>>|<cell|<math-it|Msg>,Sig<rsup|r,stage><rsub|v<rsub|i>>,v<rsub|id>>>|<row|<cell|<math-it|Msg>>|<cell|\<assign\>>|<cell|Enc<rsub|SC><around*|(|stage,V<rsup|r,stage><rsub|v>|)>>>>>
     </eqnarray*>
 
     Where:
 
     <\center>
       <tabular*|<tformat|<cwith|1|-1|1|1|cell-halign|r>|<cwith|1|-1|1|1|cell-lborder|0ln>|<cwith|1|-1|2|2|cell-halign|l>|<cwith|1|-1|3|3|cell-halign|l>|<cwith|1|-1|3|3|cell-rborder|0ln>|<cwith|1|-1|1|-1|cell-valign|c>|<table|<row|<cell|r:>|<cell|round
-      number>|<cell|unsigned 64 bit integer>>|<row|<cell|<math|id<rsub|\<bbb-V\>>>>|<cell|authority
-      set Id (Definition <reference|defn-authority-set-id>)>|<cell|unsigned
-      64 bit integer>>|<row|<cell|<right-aligned|<math|v<rsub|id>>>:>|<cell|Ed25519
-      public key of <math|v>>|<cell|32 byte
+      number>|<cell|unsigned 64-bit integer>>|<row|<cell|<math|id<rsub|\<bbb-V\>>>>|<cell|authority
+      set Id (Def. <reference|defn-authority-set-id>)>|<cell|unsigned 64-bit
+      integer>>|<row|<cell|<math|Sig<rsup|r,stage><rsub|v<rsub|i>>>>|<cell|signature
+      (Def. <reference|defn-sign-round-vote>)>|<cell|512-bit
+      array>>|<row|<cell|<right-aligned|<math|v<rsub|id>>>:>|<cell|Ed25519
+      public key of <math|v>>|<cell|256-bit
```

Comment:
I find $v_id$ for a public key very confusing here, especially since we also use $id_v$ for the index of a validator. Maybe we could come up with a better short hand, e.g. $pk_v$.

Reply:
I'll open a separate issue for this.

Reply:
Issue: #289